src/language-js/comments/handle-comments.js

```diff
@@ -755,15 +749,6 @@ function handleOnlyComments({
     return true;
   }
 
-  if (
-    followingNode?.type === "Program" &&
-    followingNode.body.length === 0 &&
-    enclosingNode?.type === "ModuleExpression"
-  ) {
-    addDanglingComment(followingNode, comment);
-    return true;
-  }
```

Comment:
This is unexpected, but somehow this can be removed, and the print result is better 😄